### PR TITLE
Update title tag, add initial metadescription.

### PR DESCRIPTION
**Pull request by omarreiss**
We are not a GUI, we are the GUI.

### CODE REVIEW

**Comment by omarreiss** [comment]
@rkoster please review the metadescription.

File: index.html [hunk 1]
```diff
@@ -55,7 +56,7 @@
         <div class="row">
           <div class="col-md-12 text-center">
             <div class="mdi-material-light-blue">
-              <h1>A GUI for BOSH</h1>
+              <h1>The GUI for BOSH</h1>
```
Comment (rkoster) [inline index.html:59]: Do we also want to use `The GUI for BOSH` on our T-Shirts instead of `GUI for BOSH`?

**Review by rkoster — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (omarreiss) [reply index.html:59]: Yes, that's a better catchphrase I think.

File: index.html [hunk 2]
```diff
@@ -5,7 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-    <title>Dozer | Managed BOSH</title>
+    <title>Dozer | The GUI for BOSH</title>
+    <meta name="description" content="Dozer offers a graphical user interface (GUI) for BOSH that makes it easy to connect, monitor and manage multiple instances of BOSH." />
```
Comment (omarreiss) [inline index.html]: To give you an idea, this is what Google will show people:

screen shot 2016-09-20 at 12 01 09

Comment (omarreiss) [reply index.html]: @rkoster ☝️

Comment (rkoster) [reply index.html]: Do we want to mention Open Source BOSH?

File: index.html (hunk 1 above)
Comment (rkoster) [reply index.html:59]: fixed in https://github.com/dozer-cloud/dozer_design/commit/49976dc5234792cd36bec09576747049f07c42d8

File: index.html (hunk 2 above)
Comment (omarreiss) [reply index.html]: Fine by me, but what extra information does that convey?

Comment (rkoster) [reply index.html]: Distinguishes from OpsManager which is essentially also a GUI for BOSH however it tries to hide BOSH while we try to make it easier to use plain BOSH (so maybe OS BOSH is not the right term).

Comment (omarreiss) [reply index.html]: Should we say something like: "Dozer offers a graphical user interface (GUI) for BOSH that replaces the BOSH CLI. It makes it easy to connect, monitor and manage multiple instances of BOSH."

Comment (rkoster) [reply index.html]: Yes, I think indeed that our goal is to replace the bosh cli.

**Comment by omarreiss** [comment]
@rkoster updated the copy. Please merge and deploy update if ok.

**Comment by rkoster** [comment]
thanks!

**Comment by omarreiss** [comment]
😜 I noticed in my open PR's and thought I'd merge it.

**Pull request merged**